dashboard/app/views/congrats/index.html.haml

```diff
@@ -1,5 +1,6 @@
 :ruby
   congrats_data = {current_user: current_user}
+  congrats_data[:english] = @is_english
```

Comment:
nit: maybe set everything in `congrats_data` the same way, either in initial `{}` or line by line, as we do in this second case.